daemon/daemon.go

```diff
@@ -161,6 +162,8 @@ func main() {
 			managementServerDown()
 			fmt.Println("==== Start Operational Mode http server if not running")
 			operationalServerUp()
+			//TODO POC avahi
+			avahi.InitMDNS()
```

Comment:
for now the operational server code is commented out, so please rebase on master.